packages/ckeditor5-utils/src/focustracker.ts

```diff
@@ -54,28 +57,102 @@ export default class FocusTracker extends /* #__PURE__ */ DomEmitterMixin( /* #_
 	public _elements: Set<Element> = new Set();
 
 	/**
-	 * Event loop timeout.
+	 * List of views with external focus trackers that contribute to the state of this focus tracker.
+	 *
+	 * @internal
+	 */
+	public _externalViews: Set<ViewWithFocusTracker> = new Set();
+
+	/**
+	 * Asynchronous blur event timeout.
 	 */
-	private _nextEventLoopTimeout: ReturnType<typeof setTimeout> | null = null;
+	private _blurTimeout: ReturnType<typeof setTimeout> | null = null;
+
+	// @if CK_DEBUG_FOCUSTRACKER // public _label?: string;
 
 	constructor() {
 		super();
 
 		this.set( 'isFocused', false );
 		this.set( 'focusedElement', null );
+
+		// @if CK_DEBUG_FOCUSTRACKER // FocusTracker._instances.push( this );
 	}
 
 	/**
-	 * List of registered elements.
+	 * List of registered DOM elements.
+	 *
+	 * **Note**: The list does do not include elements from {@link #externalViews}.
 	 */
 	public get elements(): Array<Element> {
 		return Array.from( this._elements.values() );
 	}
 
 	/**
-	 * Starts tracking the specified element.
+	 * List of external focusable views that contribute to the state of this focus tracker. See {@link #add} to learn more.
+	 */
+	public get externalViews(): Array<ViewWithFocusTracker> {
```

Comment:
I decided I didn't want to extend `#elements` with elements of external views to keep the previous API contract (avoid BCs). I came up with this property that will allow integrators to dig deeper if they want, though.